app/models/container_deployment_authentication.rb

```diff
@@ -0,0 +1,5 @@
+class ContainerDeploymentAuthentication < ApplicationRecord
```

Comment:
Why can't the existing authentications table be leveraged?  Every model can `has_many :authentications`, and I think just about everything here can reuse that except a handful of columns.  My major concern here is that we already have a model and a helper class for encrypting and decrypting passwords so they are safely handled, and creating a new table requires that you do *all* of that (which is not being done here).